prow/serviceaccounts/GoogleCloudPlatform_oss-test-infra_serviceaccounts.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
Please make this file deploy automatically.

Reply:
Done